lib/point/point.js

```diff
@@ -1,10 +1,13 @@
+// The Point class is an abstract class and as such has some empty functions
```

Comment:
Must the empty functions be replaced? If so, couldn't we have this abstract default throw an error complaining that it hasn't been overridden? Otherwise I don't mind this too much.

Reply:
Some do and some don't apparently. I have noted this as follows in all abstract classes in d4ecb7ff8016bc87b1633f2c7afbc5399b238cc3